src/core/lib/tsi/ssl_transport_security.h

```diff
@@ -137,7 +136,8 @@ tsi_result tsi_create_ssl_server_handshaker_factory(
     const size_t *pem_private_keys_sizes, const unsigned char **pem_cert_chains,
     const size_t *pem_cert_chains_sizes, size_t key_cert_pair_count,
     const unsigned char *pem_client_root_certs,
-    size_t pem_client_root_certs_size, int force_client_auth,
+    size_t pem_client_root_certs_size,
+    tsi_client_certificate_request_type client_certificate_request,
```

Comment:
The core interface is not internal... It's actually public and used by all sorts of languages some of which we don't even control. So yes, I think that I feel strongly about it :).